source/simulator/parameters.cc

```diff
@@ -528,7 +528,7 @@ namespace aspect
       prm.declare_entry ("Run postprocessors on initial refinement", "false",
                          Patterns::Bool (),
                          "Whether or not the postproccessors should be run at the end "
-                         "of each of ths initial adaptive refinement cycles at the "
+                         "of each of the initial adaptive refinement cycles at the "
                          "of the simulation start.");
```

Comment:
the grammar of that sentence wrong is as well. while you're there, can you fix this as well?